cookbook/import_export/product-import.rst

```diff
@@ -0,0 +1,450 @@
+            arguments:
+                - '@pim_connector.array_converter.flat.product'
+                - '@pim_catalog.repository.product'
+                - '@pim_catalog.builder.product'
+                - '@pim_catalog.updater.product'
+                - '@pim_catalog.validator.product'
+                - '@akeneo_storage_utils.doctrine.object_detacher'
+                - '@pim_catalog.comparator.filter.product'
+
+The class ``Pim\Component\Connector\Processor\Denormalization\ProductProcessor`` mainly delegates the operations to different technical and business services.
+
+.. code-block:: php
+
+    /**
+     * @param StandardArrayConverterInterface       $arrayConverter array converter
+     * @param IdentifiableObjectRepositoryInterface $repository     product repository
+     * @param ProductBuilderInterface               $builder        product builder
+     * @param ObjectUpdaterInterface                $updater        product updater
+     * @param ValidatorInterface                    $validator      product validator
+     * @param ObjectDetacherInterface               $detacher       detacher to remove it from UOW when skip
+     * @param ProductFilterInterface                $productFilter  product filter
+     */
+    public function __construct(
+        StandardArrayConverterInterface $arrayConverter,
+        IdentifiableObjectRepositoryInterface $repository,
+        ProductBuilderInterface $builder,
+        ObjectUpdaterInterface $updater,
+        ValidatorInterface $validator,
+        ObjectDetacherInterface $detacher,
+        ProductFilterInterface $productFilter
+    ) {
+        // ...
+    }
+
+Product Processor - StandardArrayConverterInterface
+---------------------------------------------------
+
+This service allows to transform the CSV array of items to the Standard Format array.
+
+.. code-block:: php
+
+    // CSV Format
+    $csvItem = [
+      'sku'                      => "AKNTS_BPXS"
+      'categories'               => "goodies,tshirts"
+      'clothing_size'            => "xs",
+      'description-en_US-mobile' => "Akeneo T-Shirt",
+    ];
+
+    $standardItem = $this->arrayConverter->convert($csvItem);
+
+    // Standard Format
+    [
+        'sku'           => [ 'data' => "AKNTS_BPXS", 'locale' => null, 'scope' => null ],
+        'categories'    => [ "goodies", "tshirts" ],
+        'clothing_size' => [ 'data' => "xs", 'locale' => null, 'scope' => null ]
+    ]
+
+The class ``Pim\Component\Connector\ArrayConverter\Flat\ProductStandardConverter`` provides a specific implementation to handle product data.
+
+.. note:
+
+    If you read another kind of file, xls, xml, json, etc, if you manage to convert the input array data to this format, all the other parts of the import will be reusable.
+
+.. note:
+
+    We tend to use this standard array format everywhere in the PIM, for imports, backend process, product edit form, variant group values, proposals, etc.
```

Comment:
Maybe this format could be presented in a dedicated part since it's a crucial information and we refer to it very often in the next pages. What do you think ?